src/components/SliderBanner.jsx

```diff
@@ -10,85 +12,34 @@ const SliderBanner = () => (
     data-rotation-duration={1000}
   >
     <div className="cube">
-      <section
-        className="slide"
-        style={{
-          backgroundImage: `url(${home.BannerImg1})`
-        }}
-      >
-        <div className="slide-content">
-          <h1 className="page-title">{home.BannerTitle1}</h1>
-          <div className="page-subtitle">{home.BannerSub1}</div>
-          <div className="volunteer-cta">
-            <a href="#" className="btn btn-large btn-icon js-wave">
-              <i className="material-icons">arrow_forward</i>
-              {home.BannerBtn1}
-            </a>
-            <div className="note">{home.BannerBtnSub1}</div>
+      {Banners.map(b => (
```

Comment:
why is banner in caps?

Reply:
never use 'b' as name...if your looping over something plural then just make it singluar

Reply:
destructure banner